.github/workflows/build.yml

```diff
@@ -135,17 +140,48 @@ jobs:
                 python-version: '3.7'
 
             - name: Generate build files
-              run: cmake -S. -Bbuild -A${{matrix.arch}} -DBUILD_TESTS=ON -DUPDATE_DEPS=ON
+              run: cmake -S. -B build -DBUILD_TESTS=ON -DUPDATE_DEPS=ON -D CMAKE_BUILD_TYPE=${{matrix.config}} -A ${{ matrix.arch }}
 
             - name: Build the loader
               run: cmake --build ./build --config ${{matrix.config}}
 
             - name: Run regression tests
               working-directory: ./build
-              run: ctest --output-on-failure
-
-            - name: Verify generated source files
-              run: python scripts/generate_source.py --verify external/Vulkan-Headers/registry
+              run: ctest --output-on-failure -C ${{matrix.config}}
+
+    # Something about Github Actions + Windows + Ninja + Unicode doesn't play nicely together.
+    # https://github.com/KhronosGroup/Vulkan-Loader/pull/1188#issuecomment-1536659318
+    #
+    # Disable testing explicitly for Ninja. But still ensure it builds properly.
+    windows_ninja:
```

Comment:
Unable to repro GHA unicode build failure. Instead just add a separate WIN32 ninja job which disables testing.

Reply:
I'm not sure what the issue is here. Maybe it's windows command prompt, cmake, ninja, or possibly `ilammy/msvc-dev-cmd`.

Hard to say. But it's not worth wasting any more time on. I still need to make sure the loader builds cleanly with Ninja on Windows however for VVL. So just having a seperate job with testing disabled is good enough.